packages/big-design/src/components/TableNext/DataCell/DataCell.tsx

```diff
@@ -10,7 +10,9 @@ export interface DataCellProps
     TableColumnDisplayProps,
     PaddingProps {
   align?: 'left' | 'center' | 'right';
+  checkEmptyCell?: boolean;
   children?: React.ReactNode;
+  colSpan?: number;
```

Comment:
🍹 I believe this should already be apart of `TableHTMLAttributes<HTMLTableCellElement>` 🤔

Reply:
Ohh yes, I remember ts was complaining, let me try to check it again 🤔